src/python/zquantum/optimizers/grid_search/_grid_search.py

```diff
@@ -12,48 +12,37 @@
 
 
 class GridSearchOptimizer(Optimizer):
-    def __init__(self, grid: ParameterGrid, options: Optional[Dict] = None):
+    def __init__(self, grid: ParameterGrid, recorder=_recorder):
```

Comment:
Typing needed for recorder.

Reply:
Fixed in https://github.com/zapatacomputing/z-quantum-optimizers/pull/58/commits/d4fba8b8bd7930fd68bc5a0388d153201c79bc4b